x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -751,14 +750,6 @@ export const getSavedObjectTypes = (
             },
           ],
         },
-        '15': {
```

Comment:
Just to be sure there is no risk that a project already use that version of kibana somehow?

Reply:
that's a good question. for serverless, it's not yet released. for cloud deployments, `9.0-SNAPSHOT` and `8.16-SNAPSHOT` versions may use it, but i'd say that's not a problem as they are only for testing.